### DISCUSSION TITLE
what is DiscordShardedClient

### DISCUSSION THREAD

**Discussion by Rikitav** [DSharpPlus/DSharpPlus#1656] [Support] [answered]
What is DiscordShrdedClient used for and how is it different from a simple DiscordClient?

**Comment by Plerx2493** [comment, marked as answer]
If your bot is in a certain amount of guilds, you need to use multiple gateway sessions to handle the traffic and its required from discords side if your bot reaches 1000 guilds. The DiscordShardedClient manages this sharding process for you. https://dsharpplus.github.io/DSharpPlus/articles/beyond_basics/sharding.html
